### PR TITLE
Update MUA in dropdown

### PR DESCRIPTION
**Pull request by epwinchell**
Jira: https://issues.redhat.com/browse/RHCLOUD-21403

Screen Shot 2022-11-01 at 10 25 22 AM

### CODE REVIEW

File: locales/translation-template.json [hunk 1]
```diff
@@ -79,6 +79,10 @@
     "defaultMessage": "Feedback Sent",
     "description": "Feedback Sent"
   },
+  "filterByTags": {
```
Comment (Hyperkid123) [inline locales/translation-template.json:82]: I see these changes are not related not the jira card. Are these just additional fixes?

Comment (epwinchell) [reply locales/translation-template.json:82]: @Hyperkid123 They appeared when I ran `npm run translations`  Should I remove them?

**Review by aneelac22 — APPROVED** [review]
LGTM!
